### DISCUSSION TITLE
How do I bring an instance of Airbyte running on my laptop up and down?

### DISCUSSION THREAD

**Discussion by EgorKraevTransferwise** [airbytehq/airbyte#46538] [Deployment]
I followed the instructions for starting up a local instance using abctl, and it worked like a charm; but I don't want to have it running permanently (RAM and CPU consumption), but rather bring it up and down like I do with Docker containers. 
What are the abctl commands for that?

**Comment by szemek** [comment]
There is a discussion for that https://github.com/airbytehq/airbyte/discussions/45847
For now you can stop Docker container `docker stop airbyte-abctl-control-plane`
abctl under the hood creates kind cluster https://kind.sigs.k8s.io ([kind](https://sigs.k8s.io/kind) is a tool for running local Kubernetes clusters using Docker container “nodes”.)

[a comment by an automated account is not shown]
